Orange/widgets/data/owcorrelations.py

```diff
@@ -45,17 +46,21 @@ def items():
         return ["Pearson correlation", "Spearman correlation"]
 
 
+class Cluster(SimpleNamespace):
+    instances = None  # type: Optional[List]
+    centroid = None  # type: Optional[np.ndarray]
+
+
 class KMeansCorrelationHeuristic:
     """
     Heuristic to obtain the most promising attribute pairs, when there are to
```

Comment:
to -> too
(not important, skip if you don't make any other, substantial changes)